labs/index.md

```diff
@@ -1,109 +1,58 @@
-    collaborators:
-    - name: CORE Team
-      link: https://core.ac.uk/about/#team
-    summary: |
-      Explore relevant open access papers from across a global network of
-      research repositories while browsing arXiv. Research papers are
-      recommended from both arXiv and other over 10 thousand open access data
-      providers and brought to you by
-      <a href="https://core.ac.uk/" target="_blank" rel="noopener">CORE</a>,
-      the world’s largest aggregator of open access research.
-  pwc_links:
-    title: arXiv Links to Code
-    repo: https://github.com/arXiv/arxiv-browse/tree/develop/browse/static/js/paperswithcode.js
-    thumbnail: images/pwc-logo.png
-    collaborators:
-    - name: Robert Stojnic
-      association: Papers with Code / Facebook AI Research
-    - name: Viktor Kerkez
-      association: Papers with Code / Facebook AI Research
-    - name: Ludovic Viaud
-      association: Papers with Code / Facebook AI Research
-    summary: |
-      arXiv Links to Code aims to provide an easy and convenient way to
-      find relevant code for a paper. It is using data from
-      <a href="https://paperswithcode.com" target="_blank" rel="noopener">
-      Papers with Code</a> - a free resource that links papers, code and
-      results in Machine Learning. Papers with Code is the biggest such resource and is licensed under an open license.
----
-
-{% macro render_project(project) %}
-<article class="card">
-  <div class="card-content">
-    <div class="columns">
-      <div class="column is-narrow-tablet">
-        <p class="image is-128x128" style="border: 1px solid gray"><img src="{{project.thumbnail}}" alt="" /></p>
-      </div>
-      <div class="column is-one-third-desktop is-one-half-tablet">
-        <h2>{{ project.title }}</h2>
-        <span class="label">Collaborator{%- if project.collaborators|length > 1 -%}s{%- endif -%}: </span>
-        {% for collaborator in project.collaborators %}
-        <p>{% if collaborator.link %}<a href="{{ collaborator.link }}" target="_blank" rel="noopener">{{ collaborator.name }}</a>{% else %}{{ collaborator.name }}{% endif %}<br />
-        {%- if collaborator.association -%}
-        <em>{{ collaborator.association }}</em></p>
-        {%- endif -%}
-        {% endfor %}
-      </div>
-      <div class="column">
-        <p>{{ project.summary|safe }}</p>
-      </div>
-    </div>
-    {%- if project.repo -%}
-    <p class="has-text-right"><span class="has-text-weight-bold">
-      Code: </span>{{ project.repo }}</p>
-    {%- endif -%}
-    {%- if project.more_info -%}
-    <p class="has-text-right"><span class="has-text-weight-bold">
-      More Information: </span><br/>
-      {%- for info in project.more_info -%}
-      <a href="{{ info.link}}" target="_blank" rel="noopener">{{ info.name }}</a><br/>
-      {%- endfor -%}
-    </p>
-    {%- endif -%}
-  </div>
-</article>
-{% endmacro %}
+# arXivLabs: An invitation to collaborate
```

Comment:
Is this OK at the top of the file or should it be below the html header content?

Reply:
The macro can be anywhere in the template. Its invocation can also happen just about anywhere in the HTML (as it is now) but ideally it would be in the `<head>` element.